MobileCenterAnalytics/MobileCenterAnalytics/Internals/Session/MSSessionTracker.m

```diff
@@ -105,29 +105,14 @@ - (void)start {
     }
 
     // Hookup to application events.
-    [MS_NOTIFICATION_CENTER addObserver:self
```

Comment:
I don't think you can reuse the notification observer in MobileCenter. We don't suspend any services on macOS apps but we do need to renew sessions.

Reply:
As I talked offline, the PR didn't show me the entire changes when I reviewed it. 😮 
That's why I didn't see `addObserver` call right after the line and I assumed you are reusing the one in MobileCenter class. Ignore my feedback.